clang/lib/3C/ConstraintVariables.cpp

```diff
@@ -81,12 +81,12 @@ PointerVariableConstraint *
 PointerVariableConstraint::getNamedNonPtrPVConstraint(StringRef Name,
                                                       Constraints &CS) {
   CAtoms NewVA; // empty -- represents a base type
-  return new PVConstraint(NewVA, "unsigned", Name, nullptr, false, false, "");
+  return new PVConstraint(NewVA, "unsigned", Name, nullptr, false, "");
 }
 
 PointerVariableConstraint::PointerVariableConstraint(
     PointerVariableConstraint *Ot, Constraints &CS)
-    : ConstraintVariable(ConstraintVariable::PointerVariable, Ot->BaseType,
+    : ConstraintVariable(ConstraintVariable::PointerVariable, Ot->OriginalType,
```

Comment:
This is a little surprising. Bugfix?

Reply:
Yeah, fixes a bug that's been in the code for a while I think.